notebooks/official/workbench/pricing_optimization/pricing-optimization.ipynb

```diff
@@ -402,7 +319,7 @@
         "id": "3a063f530682"
       },
       "source": [
-        "If you are using ***Vertex AI Workbench managed notebooks instance***, every cell which starts with \"#@bigquery\" will be a SQL Query. If you are using Vertex AI Workbench user managed notebooks instance or Colab it will be a markdown cell."
+        "If you're using ***Vertex AI Workbench managed notebooks instance***, every cell which starts with \"#@bigquery\" is a SQL Query. If you're using a Vertex AI Workbench user-managed notebook instance or Colab, it's a markdown cell."
```

Comment:
change to: "Vertex AI Workbench user-managed notebooks instance" (plural on "notebooks")